integrations/k8s-using-daemonset/falco-event-generator-deployment.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: falco-event-generator-deployment
+  labels:
+    name: falco-event-generator-deployment
+    app: demo
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: falco-event-generator
+    spec:
+      containers:
+      - name: falco-event-generator
+        image: sysdig/falco-event-generator:latest
```

Comment:
This needs probably to be:

1. renamed
2. coordinate with the new event-generator at https://github.com/falcosecurity/event-generator

Reply:
I think that it can be considered done :) But I added a consideration in the last comment

Reply:
Atm, `falcosecurity/event-generator` is not yet ready, although a test image already exists.

@leodido WDYT ? 
We may keep this on hold until dependant issues are solved. Regarding the `event-generator`, I guess it will take another week or so.

Reply:
Done :point_right: https://github.com/falcosecurity/event-generator/releases/tag/v0.1.0

Reply:
@leogr we'll keep the sysdig one for now? Or we can wait for a release of the Docker image v0.1.0? Thank you!

Reply:
You are welcome! 
It's released, see my review below